src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -71,7 +69,7 @@ public bool TryGetArgIndex(string argName, out int argIndex)
             return false;
         }
 
-        public TexlBinding BindBody(INameResolver nameResolver, IBinderGlue documentBinderGlue, BindingConfig bindingConfig = null, IUserDefinitionSemanticsHandler userDefinitionSemanticsHandler = null, Features features = null, INameResolver functionNameResolver = null, IExternalRule rule = null)
```

Comment:
>IUserDefinitionSemanticsHandler [](http://example.com/codeflow?start=132&length=31)

can we remove this definition completely?

Reply:
Done!